vip_spec.xsd

```diff
@@ -298,6 +298,9 @@ reference elements for polling location
         <xs:element name="Party">
           <xs:complexType>
             <xs:sequence>
+              <!-- SURVEY: 1622.2 has abbreviation as a string, not InternationalizedText.
+                   Which makes more sense?
```

Comment:
Currently VIP And 1622.2 do this differently, so absent any compelling use case I'm inclined to just go the 1622.2 compatibility route just to make it easy on election officials.